PluralKit.Bot/Commands/MemberProxy.cs

```diff
@@ -26,6 +26,7 @@ ProxyTag ParseProxyTags(string exampleProxy)
             {
                 // // Make sure there's one and only one instance of "text" in the example proxy given
                 var prefixAndSuffix = exampleProxy.Split("text");
+                if (prefixAndSuffix.Length == 1) prefixAndSuffix = prefixAndSuffix[0].Split("TEXT");
```

Comment:
hmm, is there a way to do this "nicer"? like a case-insensitive split?

Reply:
could use a regex instead, but that seems a bit overkill?

Reply:
yeah, fair.